include/ziapi/Http.hpp

```diff
@@ -17,12 +17,15 @@ struct Request {
     /// For possible values of version checkout ziapi::http::Version.
     Version version;
 
+    /// The request target contains the route and the query parameters.
+    /// The route is simply the path of the request, like `/users/profile`.
+    /// The query parameters are the parameters of the request, like `?username=toto&age=18`.
```

Comment:
I don't think ZIAPI should include documentation on the HTTP protocol. If you start explaining one thing from the protocol, you're bound to explaining everything. I believe that if a person is not familiar with the concept of a HTTP request "target" they should look it up themselves instead of relying on ZIAPI's documentation which might be untrue or incomplete.
Feel free to collect Gabriel's or Thibault's opinion on this.

Reply:
I don't think it can be qualified as documentation of the HTTP protocol. It's just an example to explicit which variables match which parts of the request.

As always, I'll let the maintainer decide, please don't just use my opinion as a "Thibaut told us to do it" joker :yum:

Reply:
I agree with Thibaut that this is in no way a documentation of the HTTP protocol.

> I believe that if a person is not familiar with the concept of a HTTP request "target" they should look it up themselves instead of relying on ZIAPI's documentation which might be untrue or incomplete.

I see two issues about this particular statement : 
- the fact that the documentation is not complete should not prevent it to actually exists. It could just be a quick remainder, with a link that redirect you to a better source of information.
- the actual definition in the RFC 2616 is "Resource-URI". As such, you cannot expect a user to "look it up", as it's not the proper terminology to begin with, and it only apply to your particular implementation. As `target` is not properly formalized, one could understand it as the `host [ ":" port ] [ abs_path]` part of the URI, without the query parameters + scheme, as they are not part of what actually is the "target" of the message, but the protocol used to communicate, as well as part of the actual message sent to that target

Now as Thibaut said, it's up to the maintainer to decide, I´m only expressing an opinion here

Reply:
Okay! Very valid feedback. Thanks!